README.md

```diff
@@ -209,6 +209,19 @@ go run examples/generate_balances/main.go \
   rosetta-cli-conf/testnet/bootstrap_balances.json
 ```
 
+### Running Rosetta with a mycelo testnet
+
+- Set `--geth.genesis` to point to the genesis file for the testnet.
+- Set `--geth.networkid` to the network ID (if this is a deployed testnet, this may be different from the `ChainID` in the genesis file). If this value is the same as the `ChainID`, it is not necessary to set this parameter.
+- Set the `--monitor.initcontracts` flag (at least on the first run), which fetches necessary state from the genesis block and updates the Rosetta DB accordingly.
+
+To run reconciliation tests on this network:
+
+- Generate `bootstrap_balances.json` using the network's genesis block.
```

Comment:
Is there a guide on how to do that? Otherwise a short explanation would be good.

Reply:
yes! there's a section in the readme above this one, just doesn't show up in the diff :)